podman/domain/containers.py

```diff
@@ -466,8 +463,8 @@ def top(self, **kwargs) -> Dict[str, Any]:
             ps_args (str): Optional arguments passed to ps
         """
         params = {
-            "ps_args": kwargs.get("ps_args", None),
-            "stream": kwargs.get("stream", None),
+            "ps_args": kwargs.get("ps_args"),
+            "stream": False,
```

Comment:
Is this right to set this to false all the time?

Reply:
IMHO yes because stream requires special handling.